src/UI/TransactionsTable/AccountName.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { Trim } from '../Trim';
+import { AssetType } from 'components/TransactionsInterpreter/helpers/types';
+
+export const AccountName = ({
+  address,
+  assets,
+  dataTestId,
+  color
+}: {
+  address: string;
+  assets?: AssetType;
+  dataTestId?: string;
+  color?: 'muted' | 'secondary';
+}) => {
+  if (assets && assets.name) {
+    const name = assets.name.replace(/\p{Emoji}/gu, '');
+    const description = `${name} (${address})`;
+    return (
+      <span
```

Comment:
you could extract to const dataTestId = {..} and then destructure below`<span {...dataTestId}`.
check out denominateComponent